install/cluster-openshift-apiserver-operator/install.yaml

```diff
@@ -0,0 +1,113 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+parameters:
+- name: NAMESPACE
+  # This namespace must not be changed.
+  value: openshift-core-operators
+objects:
+- apiVersion: v1
+  kind: Namespace
+  metadata:
+    labels:
+      openshift.io/run-level: "0"
+    name: ${NAMESPACE}
+
+- apiVersion: apiextensions.k8s.io/v1beta1
+  kind: CustomResourceDefinition
+  metadata:
+    name: openshiftapiserveroperatorconfigs.openshiftapiserver.operator.openshift.io
+  spec:
+    scope: Cluster
+    group: openshiftapiserver.operator.openshift.io
+    version: v1alpha1
+    names:
+      kind: OpenShiftAPIServerOperatorConfig
+      plural: openshiftapiserveroperatorconfigs
+      singular: openshiftapiserveroperatorconfig
+      categories:
+      - coreoperators
+    subresources:
+      status: {}
+
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: system:openshift:operator:cluster-openshift-apiserver-operator
+  roleRef:
+    kind: ClusterRole
+    name: cluster-admin
+  subjects:
+  - kind: ServiceAccount
+    namespace: ${NAMESPACE}
+    name: openshift-cluster-openshift-apiserver-operator
+
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    namespace: ${NAMESPACE}
+    name: openshift-cluster-openshift-apiserver-operator-config
+  data:
+    config.yaml: |
+      apiVersion: operator.openshift.io/v1alpha1
+      kind: GenericOperatorConfig
+
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    namespace: ${NAMESPACE}
+    name: openshift-cluster-openshift-apiserver-operator
+    labels:
+      app: openshift-cluster-openshift-apiserver-operator
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app: openshift-cluster-openshift-apiserver-operator
+    template:
+      metadata:
+        name: openshift-cluster-openshift-apiserver-operator
+        labels:
+          app: openshift-cluster-openshift-apiserver-operator
```

Comment:
should this have the checkpointer labels if we want to test the restore flow at some point?

Reply:
> should this have the checkpointer labels if we want to test the restore flow at some point?

no.  openshift-apiserver isn't checkpointed.